precompiles/randomness/Randomness.sol

```diff
@@ -1,81 +1,152 @@
 // SPDX-License-Identifier: GPL-3.0-only
 pragma solidity >=0.8.0;
 
+/// @dev Maximum number of random words being requested
+uint32 constant MAX_RANDOM_WORDS = 100;
```

Comment:
will this always be less than 2^8 - 1 = 255? I'm thinking of using `u8` for `num_words` field on the Request struct. 

It doesn't really matter, I can make it u32, just asking to make sure

Reply:
Yes it doesn't really matter as it will always be encoded into a u256 IIRC